app/src/main/java/com/example/concordia_campus_guide/Fragments/InfoCardFragment/InfoCardFragmentViewModel.java

```diff
@@ -1,36 +1,32 @@
 package com.example.concordia_campus_guide.Fragments.InfoCardFragment;
 
-import android.widget.TextView;
-import androidx.lifecycle.ViewModel;
+import android.app.Application;
 
+import androidx.annotation.NonNull;
+import androidx.lifecycle.AndroidViewModel;
+
+import com.example.concordia_campus_guide.Database.AppDatabase;
 import com.example.concordia_campus_guide.Models.Building;
+import com.example.concordia_campus_guide.Models.Place;
 
-public class InfoCardFragmentViewModel extends ViewModel {
+public class InfoCardFragmentViewModel extends AndroidViewModel {
```

Comment:
Why did you change the extend AndroidViewModel instead of ViewModel? Any particular reason behind this choice ?

Reply:
Need constructor for db, that comes with AndroidViewModel

Reply:
ok, I see thank you for clarifying that 👍